common/src/service/HttpClientHelper.cs

```diff
@@ -700,19 +721,34 @@ private static void InsertEtag(HttpRequestMessage requestMessage, string etag)
         {
             cancellationToken.ThrowIfCancellationRequested();
 
-            using (var httpClient = BuildHttpClient(Timeout.InfiniteTimeSpan))
+            HttpClient httpClient = null;
+            HttpClientHandler httpClientHandler = null;
+
+            try
             {
-                var cts = new CancellationTokenSource(operationTimeout);
-                CancellationTokenSource linkedCts = CancellationTokenSource.CreateLinkedTokenSource(cancellationToken, cts.Token);
-                await ExecuteAsync(
-                    httpClient,
-                    httpMethod,
-                    requestUri,
-                    modifyRequestMessageAsync,
-                    isMappedToException,
-                    processResponseMessageAsync,
-                    errorMappingOverrides,
-                    linkedCts.Token).ConfigureAwait(false);
+                (httpClient, httpClientHandler) = BuildHttpClient(Timeout.InfiniteTimeSpan);
```

Comment:
I should have probably pointed this out too :) Same suggestion as above for using.